launcher/launcher.go

```diff
@@ -442,14 +303,14 @@ func (launcher *Launcher) RestartInstances() error {
 	launcher.Lock()
```

Comment:
RestartInstances will contains list of instances.

Reply:
But why? `RestartInstances` is used when unitconfig is updated to restart already started instances. What is the point to use RestartInstances with list of instances?

Reply:
After unit config changes rebalanicig will be performed and list of instances could be changed